x/stake/types/delegation.go

```diff
@@ -33,29 +33,30 @@ func (d Delegation) GetDelegator() sdk.Address { return d.DelegatorAddr }
 func (d Delegation) GetValidator() sdk.Address { return d.ValidatorAddr }
 func (d Delegation) GetBondShares() sdk.Rat    { return d.Shares }
 
-//Human Friendly pretty printer
+// HumanReadableString returns a human readable string representation of a
+// Delegation. An error is returned if the Delegation's delegator or validator
+// addresses cannot be Bech32 encoded.
 func (d Delegation) HumanReadableString() (string, error) {
```

Comment:
We can probably just call this `String()` right @rigelrozanski  ?

Reply:
Agreed -- this should just implement the `Stringer` interface.

Reply:
Actually, this will require a bit more work that I think is outside the scope of this ticket (since it also returns an error and the Stringer interface does not). I'll create another ticket for this.